pending/biglybt.xml

```diff
@@ -12,32 +12,43 @@
 
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.
 
     You should have received a copy of the GNU General Public License
-    along with this program. If not, see <http://www.gnu.org/licenses/>.
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app BiglyBT
     @url https://www.biglybt.com/
+    @os Windows, Linux
     @cleanerversion v2.0.0
-    @cleanerdate 2019-03-11
+    @cleanerdate 2019-03-14
     @cleanerby https://github.com/Tobias-B-Besemer
-    @tested ok 
-    @testeddate 
-    @testedby 
-    @note 
+    @tested ok v???, Windows 7
+    @testeddate ???
+    @testedby https://github.com/Tobias-B-Besemer
+    @note Cleaner was forged from vuze.xml originaly done by Andrew Ziem
+    @note Linux $ProgramFiles not yet figured out! I will do this later! Tobias.
+    @note BiglyBT AzConfigBackup not yet figured out! I will do this later! Tobias.
+    @note Will check for ._az & ._az.saving files later! Tobias.
 
 -->
 <cleaner id="biglybt">
   <label>BiglyBT</label>
   <description>File transfer client</description>
   <var name="profile">
-    <value search="glob" os="windows">$APPDATA\BiglyBT</value>
+    <value search="glob" os="windows">$AppData\BiglyBT</value>
     <value search="glob" os="linux">~/.biglybt</value>
   </var>
   <var name="programfiles">
-    <value search="glob" os="windows">$ProgramFiles\BiglyBT</value>
-    <!-- Not yet figured out! I will do this later! Tobias
+    <!--
+    Windows itself changes %ProgramFiles% for 32-bit processes running in a 64-bit operating system as documented here:
+    https://docs.microsoft.com/en-us/windows/desktop/winprog64/wow64-implementation-details
+    ...so we use %SystemDrive%\Program Files\...
+    -->
+    <value search="glob" os="windows">%SystemDrive%\Program Files\Vuze</value>
```

Comment:
`ProgramW6432` is better than `%SystemDrive%\Program Files`

Reply:
I guess that Win32 don't know %ProgramW6432%...

Reply:
Do you mean the variable ProgramW6432 is not defined?

Reply:
Not on 32-bit systems as far as I understand it!

Reply:
I can use it, but I don't know what BleachBit does if you run it on W32 and `%ProgramW6432%` doesn't exist...

Reply:
If the environment variable doesn't exist, the path will be invalid and not exist. Then it will quietly be skipped. This happens all the time. It can happen when a path refers to another operating system, Linux vs Windows, for example. Try it as an experiment. Make up a variable or path that doesn't exist

Reply:
I already solved it - PR follows!